internal/provisioner/helm_utils.go

```diff
@@ -26,7 +26,7 @@ type helmDeployment struct {
 	namespace           string
 	setArgument         string
 	valuesPath          string
-	desiredVersion      string
+	desiredVersion      model.UtilityVersion
```

Comment:
Do we still need `valuesPath` if we have `ValuesPath()` method on the `desiredVersion model.UtilityVersion`?

Reply:
Possibly not, good catch.

Reply:
You're correct here too, thanks!